nextcord/guild.py

```diff
@@ -3204,8 +3208,8 @@ async def create_scheduled_event(
         privacy_level: ScheduledEventPrivacyLevel = ScheduledEventPrivacyLevel.guild_only,
         end_time: datetime.datetime = MISSING,
         description: str = MISSING,
-        image: bytes = MISSING,
-        reason: Optional[str] = None,
+        image: Union[bytes, Asset, Attachment] = None,
+        reason: Optional[str] = None
```

Comment:
Trailing commas are good. For one thing, it allows changing fewer lines when adding parameters.

```suggestion
        reason: Optional[str] = None,
```